src/main/java/uk/gov/hmcts/reform/ccd/document/am/service/impl/CaseDataStoreServiceImpl.java

```diff
@@ -69,23 +71,23 @@ public CaseDataStoreServiceImpl(RestTemplate restTemplate, SecurityUtils securit
                 CaseDocumentMetadata caseDocumentMetadata = new ObjectMapper().convertValue(responseObject.get("documentMetadata"),
                                                                                             CaseDocumentMetadata.class);
                 if (null == caseDocumentMetadata.getDocument()) {
-                    LOG.error("Could't find document for case  : " + caseId + ", response code from CCD : " + HttpStatus.FORBIDDEN);
-                    throw new ForbiddenException("Could't find document for case  : " + caseId);
+                    LOG.error(ERROR_MESSAGE, caseId, HttpStatus.FORBIDDEN);
+                    throw new ForbiddenException(CASE_ERROR_MESSAGE + caseId);
                 }
                 return Optional.of(caseDocumentMetadata);
             }
         } catch (HttpClientErrorException exception) {
             if (HttpStatus.NOT_FOUND.equals(exception.getStatusCode())) {
-                LOG.error("Could't find document for case  : " + caseId + ", response code from CCD : " + HttpStatus.NOT_FOUND);
-                throw new ForbiddenException("Could't find document for case  : " + caseId);
+                LOG.error(ERROR_MESSAGE, caseId, HttpStatus.NOT_FOUND);
+                throw new ForbiddenException(CASE_ERROR_MESSAGE + caseId);
```

Comment:
We are checking 404(Not Found) and throwing 403 (Forbidden) Exception.It looks very weird.

Reply:
Valid comment and can be incorporated in functional test task.